files/fluentd/fluentd_prometheus_alerts.yaml

```diff
@@ -31,13 +31,26 @@
     "labels":
       "service": "fluentd"
       "severity": "critical"
-  - "alert": "FluentdErrorsHigh"
-    "annotations":
-      "message": "In the last minute, {{ $value }} errors reported by fluentd {{ $labels.instance }}."
-      "summary": "Fluentd reports high number of errors"
-    "expr": |
-      sum by(instance, job) (rate(fluentd_output_status_num_errors[1m])) > 10
-    "for": "1m"
-    "labels":
-      "service": "fluentd"
-      "severity": "critical"
+
+  - alert: FluentDHighErrorRate
+    annotations:
+      message: |-
+        FluentD has a high error rate
```

Comment:
so before we had more information in our messages, why do you feel its unnecessary?

Reply:
I can reuse the more verbose message. That part was unintentional.

Reply:
do we have the same sorting as we had before where we can be specific to a fluentd instance? (generally curious)

Reply:
Yes, because the instance label is still included in the query (I've adjusted it again)